pkg/kubernetes/kubernetes.js

```diff
@@ -125,6 +125,24 @@ define([
             console.warn(ex);
         }
 
+        this.delete_pod = function delete_pod(pod_name) {
+            api.request({"method": "DELETE",
+                "params": "",
```

Comment:
I think that specifying an empty params is unnecessary. It'll be empty by default. Suggest removing this line.